### PR TITLE
New layout for integration config

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Needs some cleanup and translations still.

[Preview screencapture](https://cdn.discordapp.com/attachments/351047592588869643/701821233032855592/config-integrations.mp4)

image

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-related-items.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ export class HaRelatedItems extends SubscribeMixin(LitElement) {
                 )}:
               </h3>
               <a
-                href="/config/integrations/config_entry/${relatedConfigEntryId}"
+                href="/config/integrations#${relatedConfigEntryId}"
```
Comment (balloob) [inline src/components/ha-related-items.ts:98]: Let's start using query params

```suggestion
                href="/config/integrations#configEntry=${relatedConfigEntryId}"
```

**Review by balloob — APPROVED** [review]

**Comment by SeanPM5** [comment]
Awesome! 🎉 

Minor visual nitpick, I think it'd look a bit cleaner by removing the underline on the links ("X devices and X entities" part). The underlines make it a little harder to read IMO and you can already tell they're links based on the color.

**Comment by timmo001** [comment]
@SeanPM5 This could be done locally, but would probably make more sense to change all links to follow this style (no underline) by default otherwise it would introduce a design inconsistency IMHO.

Maybe a topic for somewhere else though :smiley:

File: src/components/ha-related-items.ts [hunk 2]
```diff
@@ -95,7 +96,7 @@ export class HaRelatedItems extends SubscribeMixin(LitElement) {
                 )}:
               </h3>
               <a
-                href="/config/integrations#${relatedConfigEntryId}"
+                href="/config/integrations?config_entry=${relatedConfigEntryId}"
```
[the remark this answers is not in the document]
Comment (bramkragten) [reply src/components/ha-related-items.ts:99]: Oh, figgered that was a typo as a query param is with the `?` right? This would be a hash param :-)

**Pull request merged**